src/app/shared/services/user.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable, NgZone } from '@angular/core';
-import { User } from "../services/user";
+import { User } from "./user/user";
```

Comment:
As for my code this one should be like following
import { User } from "./user";

Reply:
please pull again. I have changed the structure